tests/sql/moz-fx-data-shared-prod/account_ecosystem_derived/ecosystem_user_id_lookup_v1/test_script/telemetry_stable.account_ecosystem_v4.yaml

```diff
@@ -1,4 +1,5 @@
-- submission_timestamp: 2020-04-01 22:09:03.243074 UTC
```

Comment:
These test files need to present a yaml list of documents. In this case, it's only a single document, but needs to be preceeded by a `-` to make sure it's a list rather than having all these fields at the top level. The next file has the same issue.